README.md

```diff
@@ -213,20 +217,33 @@ docker logs jupyterhub
 
 ### How do I specify the Notebook server image to spawn for users? 
 
-In this deployment, JupyterHub uses DockerSpawner to spawn single-user Notebook servers.  The `jupyterhub_config.py` reads the image name from the `DOCKER_CONTAINER_IMAGE` environment variable.
+In this deployment, JupyterHub uses DockerSpawner to spawn single-user
+Notebook servers. You set the desired Notebook server image in a
+`DOCKER_CONTAINER_IMAGE` environment variable.
+
+When spawning single-user notebooks, JupyterHub reads the DockerSpawner's 
+image name from `jupyterhub_config.py` file. This file includes the
+DockerSpawner's image name which refers to desired image in the
+`DOCKER_CONTAINER_IMAGE` environment variable:
 
```

Comment:
I get tripped up a bit on this one.  It reads like the `jupyterhub_config.py` is consulted when an image is spawned, but the `DOCKER_CONTAINER_IMAGE` is only read during JHub startup.  And one could interpret "DockerSpawner's image name" as the spawner itself having an image, but we're really talking about the Notebook server image that DockerSpawner uses to spawn a container.  

Perhaps: "JupyterHub reads the Notebook image name from `jupyterhub_config.py`.  This file retrieves the Notebook image name from the `DOCKER_CONTAINER_IMAGE` environment variable:"

I think the name of the variable may be causing some confusion.  Perhaps we should name it something like, `DOCKER_NOTEBOOK_IMAGE`, or something.